pkg/appconsts/versioned_consts.go

```diff
@@ -26,7 +27,17 @@ func SquareSizeUpperBound(_ uint64) int {
 	return v1.SquareSizeUpperBound
 }
 
+func TxSizeCostPerByte(_ uint64) uint64 {
+	return v3.TxSizeCostPerByte
+}
+
+func GasPerBlobByte(_ uint64) uint32 {
+	return v3.GasPerBlobByte
+}
+
 var (
 	DefaultSubtreeRootThreshold = SubtreeRootThreshold(LatestVersion)
 	DefaultSquareSizeUpperBound = SquareSizeUpperBound(LatestVersion)
+	DefaultTxSizeCostPerByte    = TxSizeCostPerByte(LatestVersion)
+	DefaultGasPerBlobByte       = GasPerBlobByte(LatestVersion)
```

Comment:
Blocking: these constants aren't defined for v2 and `LatestVersion = v2.Version` so these are broken.

Reply:
I only did that because we're about to upgrade the app to v3 and I believe no matter what you pass into the function as a version it will return v3.GasPerBlobByte or v3.TxSizeCostPerByte

Reply:
This convo can be resolved bc LatestVersion = 3.